zebra/lib/zebra/apis/public_job_api/serializer.ex

```diff
@@ -27,6 +27,17 @@ defmodule Zebra.Apis.PublicJobApi.Serializer do
   end
 
   def spec(job) do
+    # for restricted jobs we do not want to expose their environment
+    if job.spec["restricted_job"] do
```

Comment:
Would it make more sense to hide those jobs completely in the API?

Reply:
They are occupying resources and quotas and are not visible in UI except in the Activity monitor while running, so I'm not sure if it is ok to hide them completely. 🤔

Reply:
Fair enough